packages/yarnpkg-pnp/sources/loader/applyPatch.ts

```diff
@@ -316,6 +316,7 @@ export function applyPatch(pnpapi: PnpApi, opts: ApplyPatchOptions) {
     if (requireStack.length > 0)
```

Comment:
Is this still needed when we capture a stack trace right below it?
(Added in https://github.com/yarnpkg/berry/pull/613)

Reply:
The require stack (different from the call stack) is still useful in itself to see which files required which other, since there's at least two padding calls between each require. Plus it's also a core Node feature.